eclipse-collections-code-generator/src/main/resources/impl/map/immutable/immutableObjectPrimitiveHashMap.stg

```diff
@@ -328,6 +331,23 @@ final class ImmutableObject<name>HashMap\<K> extends AbstractImmutableObject<nam
         return this.delegate.keyValuesView();
     }
 
+    <if(!primitive.booleanPrimitive)>
+    @Override
+    public Immutable<name>ObjectMap\<K> flipUniqueValues()
+    {
+        Mutable<name>ObjectMap\<K> result = <name>ObjectMaps.mutable.empty();
+        this.forEachKeyValue((key,value)->
```

Comment:
CheckStyle violation here.

Reply:
Sorry, what is the violation? I am not able to figure it out.

Reply:
Whitespace after comma.

Reply:
Good catch. Fixed!